### ISSUE TITLE
Request for UCR Dataset Download in CSV Format

### ISSUE THREAD

**Issue by dxiaosa** [sintel-dev/Orion#530]
Hi, I'm interested in how to obtain the UCR dataset that was used in benchmark experiments in CSV format. Could someone please provide guidance on where I can download this dataset? 

Thanks.

**Comment by dxiaosa** [comment]
Thank you for your guidance @sarahmish !

When I download the dataset and open the dataset within folder 'UCR_Anomaly_FullData', all the files are followed by txt
 extension like shown in screenshot.  But when loading signal in our benchmark.py, the filename hints at csv file, for example, '183-qtdbSel100MLII-train.csv'. So, do we need to do some preprocessing here? Thanks.

![Screenshot 2024-04-17 at 4 09 23 PM](https://github.com/sintel-dev/Orion/assets/17454419/5f454070-9922-46a6-8ad9-31c51b71adc4)

**Comment by sarahmish** [comment]
Yes, the data needs to be preprocessed to Orion format. Here is a [link to a notebook](https://colab.research.google.com/drive/1QsYjUX3DgS8ccvDtxEgLeHHmbtPViIqV?ouid=101325702792223253477&usp=drive_link) that helps you achieve that.

**Comment by dxiaosa** [comment]
That works and really appreciate @sarahmish

**Comment by sarahmish** [comment]
Perfect! Closing this issue
